config/dockerfiles/Dockerfile.Rust

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
Okay… from a quick look over on Jimmy Cuadra's history his skill set is in my wheel house but he has more dev-ops  experience and Bachelors degree.  He's been blogging quality posts for about 9 years.  And he's been a good contributor to the open source community for quite some time.  He's been publishing this Rust Docker container responsibly and minimally for 2 years.

He has a good track record and is very responsible thus far with this repo.  He sounds like an okay source to trust.  What do you say @yarmiganosca, shall we give his docker image our approval as the primary maintainer?